### PR TITLE
Add controller charm units to new HA controller machines

### PR DESCRIPTION
**Pull request by wallyworld**
For new controller machines created by enable-ha, add a controller charm unit to those machines.

Update the controller charm set up to write values for:
- controller-url
- model-url-template
(these will be used by the dashboard charm).

## QA steps

cd charms
git clone https://github.com/juju/juju-controller.git
cd juju-controller
charmcraft build

$ juju bootstrap lxd --controller-charm /path/to/charms/juju-controller/build --no-default-model
$ juju enable-ha
$ juju status

see each machine has a controller charm

$ juju add-machine
$ juju enable-ha -n 5 --to 3
$ juju statuus

see each machine has a controller charm

### CODE REVIEW

**Comment by wallyworld** [comment]
[controller charm units added to HA controllers](https://trello.com/c/tawKkXOm/1332-controller-charm-units-added-to-ha-controllers)

File: state/enableha.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 	"strconv"
 
 	"github.com/juju/errors"
+	"github.com/juju/juju/environs/bootstrap"
```
Comment (benhoyt) [inline state/enableha.go:11]: Should the juju/juju imports be in a separate group at the bottom? (For consistency with other files)

**Review by benhoyt — APPROVED** [review]
LGTM

File: state/enableha.go (hunk 1 above)
Comment (wallyworld) [reply state/enableha.go]: yup, fixed

**Comment by wallyworld** [comment]
$$merge$$

**Pull request merged**